JSON_Schemas/Weights_and_Condition_Scores/iscFrameScore.json

```diff
@@ -0,0 +1,29 @@
+{
+    "required": [
+    ],
+
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "iscScoreEventCore.json"
+        },
+        {
+            "properties": {
+                "frameScoreReference": {
+                    "type": "object",
+                    "$ref": "iscScoreReference.json",
+                    "description": "The scheme or scoring reference guideline being used"
+                },
+                "score":{
+                    "type": "number",
+                    "description": "Value of the frame score"
```

Comment:
Add to description - Frame score is a  convenient way to discuss and evaluate height is in units of 'frame score' based on the height over the hips at a given age

Reply:
Added the description